components/maker/styled.js

```diff
@@ -11,3 +11,43 @@ export const SidebarButton = Styled.div`
     background-color: ${({ active }) => active ? '#201f6d26' : '#201f6d0d'};
   }
 `
+
+export const MakerCardHeader = styled.h2`
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 28px;
+  color: #0F1117;
+  margin-bottom: 16px;
+`
+
+export const MakerCardSubheader = styled.h3`
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  color: #0F1117;
+  margin-bottom: 8px;
+`
+
+export const MakerCardText = styled.h2`
```

Comment:
When we create re-usable typography, it's best to extend `Type` from `blockstack-ui` and add extra styles to it. That way, we keep all some of the existing typography elements from our design system, and only change what we need to.

Semantically, if you want an `h2`, you can use `Type.h2`. To extend it, just do:

~~~javascript
styled(Type.h2)`
  font-size: 14px
`
~~~

And, you actually don't need to write any CSS to do what you're doing here. It'd be better to create a component that renders the `Type` component with the existing props we have setup. So, you can do:

~~~javascript
const MakerCardText = ({ ...rest }) => <Type.h2 fontSize={2} color="#677282" /> // etc for other props
~~~

Also, since we use `styled-system`, we tend to stay away from hard-coded things like font sizes. We have a sizing scale built in, so you can specify the numbers 1-7, 1 being the smallest. The same goes for things like padding and margins.

cc @aulneau if he has other thoughts, and Thomas if you could point to resources around how `styled-system`'s spacing works, that'd be helpful (I couldn't find it)

Reply:
Duly noted. I need to read the resources Thomas has listed in the blockstack-ui repo still.